gammapy/estimators/metadata.py

```diff
@@ -0,0 +1,256 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import logging
+from typing import Optional
+import numpy as np
+from scipy import stats
+from astropy.coordinates import SkyCoord
+from pydantic import ValidationError, validator
+from gammapy.utils.metadata import CreatorMetaData, MetaData
+
+__all__ = ["FluxMetaData"]
+
+SEDTYPE = ["dnde", "flux", "eflux", "e2dnde", "likelihood"]
+FPFORMAT = ["gadf-sed", "lightcurve"]
+STANDARD_KEYS = [
```

Comment:
I would propose to handle this in a more generic way. Introduce a dict that maps the meta data attribute to its FITS name:

```python
FITS_META_KEYS = {
    "sed_type": "SED_TYPE",
    "instrument": "INSTRU",
...
}
```
You can also introduce the inverse mapping and the later use it avoid all the hard coded `if` statements.

Reply:
I like it a lot. Thanks for the proposal... knowing that we are already at the edge with serialisation.